components/automate-ui/src/app/pages/project/rules/project-rules.component.spec.ts

```diff
@@ -175,47 +163,74 @@ describe('ProjectRulesComponent', () => {
       component.ruleForm.get('conditions').setValue([
         {
           attribute: 'CHEF_ORGS',
-          operator: 'EQUAL',
+          operator: <ConditionOperator>'EQUALS',
```

Comment:
I found a flaw in TypeScript! No, just a flaw in my understanding. Adding the type cast here does not do anything! That is, it would be perfectly happy with `<ConditionOperator>'foobar'` and I do not see why it does not cause a compilation error. However, to me it is still valuable to put the type cast here: it tells me loud and clear where this comes from (and if you hover over `ConditionOperator` it tells you the allowed value, too.